backport-Guide.md

```diff
@@ -0,0 +1,84 @@
+# Kata Containers stable backport workflow
+
+* [Backport Workflow](#backport-workflow)
+
+This document provides a short guide on how to perform a stable backport of a
+PR or patchset in the Kata Containers repositories.
+
+It does not cover all eventualities that may be encountered whilst performing
+a backport, and only documents one potential git based workflow, but should
+serve as a short introduction and reminder of the basic process.
+
+## Backport Workflow
+
+The basic workflow involves creating a new local branch from the stable tree you
+are targetting, then cherry picking the commits from your master branch PR
+onto that branch. You then submit your branch to github as a PR against the
+stable branch (and not the `master` branch).
+
+The following example shows a pseudo-backport of a PR from the `master` branch
+into the `stable-1.2` branch of the `runtime` repo.
+
+- Ensure your local repo is upto date:
+
+```bash
+$ cd ${GOPATH}/src/github.com/kata-containers/runtime
+$ git fetch upstream
+```
+
+- Create your branch to work on
+```bash
+$ git checkout -b my_1.2_pr_backport origin/stable-1.2
+```
+
+- Locate the commits you wish to pull in
+
+Here we are looking in the `master` branch, presuming the PR has already been merged,
+but if you have the PR in a local branch you can substitute `master` for the name
+of that branch here.
+
+```bash
+$ git log --oneline master
+# And search for the SHA of the commits you wish to backport.
+```
+
+- Pull in your commits
+
+If you are pulling them in from the `master` branch, then you can add the `-x`
+argument to `git cherry-pick` to automatically add a reference in the
+commit to the original commits. This is strongly recommended to aid traceability.
+If you are picking the commits from your local branch tnen do *not* use `-x`, as this
```

Comment:
Lines 50-52 suggested rewrite: 
If you pick the commits from your local branch do *not* use `-x`. This potentially adds references to SHAs that only exist in your user branches, which is not useful for future tracability.